### DISCUSSION TITLE
Google Assistant - error message registering a device

### DISCUSSION THREAD

**Discussion by amontali** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#951] [General]
Hi, I'm very new in these things .. I was tryin to add Google Assistant on my PC following a YouTube tutorial.
In Actions Console I created my project but I have an error in the"device registration" step.
This is the error:
"An error occurred enabling Actions API. To use Actions SDK, enable the Actions API on Google Cloud Platform"

I enterd in Google Cloud and activated Actions API for the mentioned project, but nothing changes.
The error is still there .. 

Someone can help me please?

Thanks  :-)

**Comment by amontali** [comment]
If I start building the project anyway, through this screen 
![image](https://github.com/user-attachments/assets/81c7544a-f63c-40ce-9df5-6d4a71e74fa8)

... and then I go to "Device registration", and I register the device, a different error is displayed:
![image](https://github.com/user-attachments/assets/84f894b2-f31e-48d4-b415-069b0df20e67)
![image](https://github.com/user-attachments/assets/02fd3361-1280-4650-bb5b-94a4b8a53f53)

...but I only have 1 registerd model in the list ...
![image](https://github.com/user-attachments/assets/c8b79282-9f8c-4cae-9e25-cb3accda5e8a)
![image](https://github.com/user-attachments/assets/905a0d19-d3dc-408c-88d0-d3f9e52de4ca)

.. and this probably is and incomple registration  .. 

Dont know what to do .. 
Waiting for help from above ;-))

**Reply by MicroBitv2** [reply]
I have the same thing as while

**Reply by im-coder-lg** [reply]
@amontali you have the device "registered", have you carried out the process from there?
Try that, I'd like to know what happens next

**Reply by im-coder-lg** [reply]
To be honest, I think Google is sunsetting Actions Console, judging by this issue. Not surprising at all.
